heartbeat/heartbeat.reference.yml

```diff
@@ -714,7 +713,7 @@ output.elasticsearch:
   # using any event field. To set the topic from document type use `%{[type]}`.
   #topic: beats
 
-  # The Kafka event key setting. Use format string to create unique event key.
+  # The Kafka event key setting. Use format string to create an unique event key.
```

Comment:
Same comment as earlier. Do a global search to change "an unique key" to "a unique key".